OsmAnd/src/net/osmand/plus/myplaces/tracks/dialogs/AvailableTracksFragment.java

```diff
@@ -515,11 +516,15 @@ public void onSmartFoldersUpdated() {
 	}
 
 	@Override
-	public void showChangeAppearanceDialog(@NonNull SmartFolder folder) {
-		selectionHelper.setAllItems(folder.getTrackItems());
-		selectionHelper.setSelectedItems(folder.getTrackItems());
-		selectionHelper.setOriginalSelectedItems(folder.getTrackItems());
-		super.showChangeAppearanceDialog(folder);
+	public void showEditFiltersDialog(@NonNull SmartFolder folder) {
+		FragmentManager manager = getFragmentManager();
+		ArrayList<TrackItem> trackItems = new ArrayList<>();
+		trackItems.addAll(smartFolderHelper.getAllAvailableTrackItems());
```

Comment:
Mo readable code will be List<TrackItem> items = new ArrayList<>(smartFolderHelper.getAllAvailableTrackItems());